src/Binding/JSSystem.cpp

```diff
@@ -25,48 +25,21 @@
 namespace Nidium {
 namespace Binding {
 
-// {{{ Preamble
-static bool
-nidium_system_getOpenFileStats(JSContext *cx, unsigned argc, JS::Value *vp);
-
-static JSClass system_class = { "System",
-                                0,
-                                JS_PropertyStub,
-                                JS_DeletePropertyStub,
-                                JS_PropertyStub,
-                                JS_StrictPropertyStub,
-                                JS_EnumerateStub,
-                                JS_ResolveStub,
-                                JS_ConvertStub,
-                                nullptr,
-                                nullptr,
-                                nullptr,
-                                nullptr,
-                                nullptr,
-                                JSCLASS_NO_INTERNAL_MEMBERS };
-
-static JSFunctionSpec system_funcs[] = { JS_FN("getOpenFileStats",
-                                               nidium_system_getOpenFileStats,
-                                               0,
-                                               NIDIUM_JS_FNPROPS),
-#if 0
-#ifdef NIDIUM_PRODUCT_UI
-    JS_FN("language", nidium_nidium_language, 0, NIDIUM_JS_FNPROPS),
-#endif
-#endif
-                                         JS_FS_END };
-// }}}
-
 // {{{ Implementation
-static bool
-nidium_system_getOpenFileStats(JSContext *cx, unsigned argc, JS::Value *vp)
+JSSystem *JSSystem::Constructor(JSContext *cx, JS::CallArgs &args,
```

Comment:
::Constructor implementation can be removed now. (I recently default'ed it to ClassMapper).